usr/share/rear/layout/prepare/GNU/Linux/110_include_lvm_code.sh

```diff
@@ -110,11 +110,16 @@ elif lvm vgcfgrestore --force -f "$VAR_DIR/layout/lvm/${vg}.cfg" $vg >&2 ; then
         [ \$vg == "$vg" ] || continue
         # Consider Thin Pools only
         [[ ,\$layout, == *,thin,* ]] && [[ ,\$layout, == *,pool,* ]] || continue
-        # Remove twice, the first time it fails because of thin volumes in the pool
-        lvm lvremove -q -f -y \$vg/\$lv || true
-        lvm lvremove -q -f -y \$vg/\$lv
+        # Use "--force" twice to bypass any error due to invalid transaction id
+        lvm lvremove -q -f -f -y $vg/\$lv
     done
 
+    # Once Thin pools have been removed, we can activate the VG
+    lvm vgchange --available y $vg >&2
```

Comment:
Just a question, how/when was VG activated before ? Was it just omitted ?

Reply:
Yes, was missing.